ai-services/internal/pkg/validators/servicereport/servicereport.go

```diff
@@ -0,0 +1,37 @@
+package servicereport
+
+import (
+	"github.com/project-ai-services/ai-services/internal/pkg/cli/helpers"
+	"github.com/project-ai-services/ai-services/internal/pkg/constants"
+	"k8s.io/klog/v2"
+)
+
+type ServiceReportRule struct{}
+
+func NewServiceReportRule() *ServiceReportRule {
+	return &ServiceReportRule{}
+}
+
+func (r *ServiceReportRule) Name() string {
+	return "servicereport"
```

Comment:
not sure if we have added this entry anywhere in the help message so that user can skip if need be

Reply:
But do we want this to be skipped? its crucial validation and skipping may lead to lot of weird issues IMO

Reply:
skip validation is a generic, any check can be skipped anyways! if somebody is skipping then they know what they are doing it.